lib/matplotlib/ticker.py

```diff
@@ -2545,7 +2545,7 @@ def __call__(self):
         if vmin > vmax:
             vmin, vmax = vmax, vmin
 
-        if len(majorlocs) > 0:
+        if len(majorlocs) >= 2:
```

Comment:
can you just do an early `return []` around line 2522 instead of having to handle this case throughout the function?